conjure/juju.py

```diff
@@ -300,6 +307,8 @@ def do_add_all():
         for a, b in list(relations):
             params = {"Endpoints": [a, b]}
             try:
+                if msg_cb:
+                    msg_cb("Setting application relations")
```

Comment:
do we want this to say which relation? If not, let's just put it outside the 'for a,b in list(relations)' loop